packages/openai/src/openai-chat-language-model.ts

```diff
@@ -613,6 +629,7 @@ export class OpenAIChatLanguageModel implements LanguageModelV1 {
                 promptTokens: usage.promptTokens ?? NaN,
                 completionTokens: usage.completionTokens ?? NaN,
               },
+              ...(providerMetadata != null ? { providerMetadata } : {}),
```

Comment:
You can just pass in `providerMetadata`, see Anthropic: https://github.com/vercel/ai/blob/main/packages/anthropic/src/anthropic-messages-language-model.ts#L419

Reply:
Hmm this actually causes tests to fail due to a `toStrictEqual` assertion - it makes the providerMetadata present when it shouldn't be.